09/tic-tac-toe.instructor.md

```diff
@@ -51,17 +57,17 @@ def tic_tac_toe_winner(board):
       ((0,0), (0,1), (0,2)),
       ((1,0), (1,1), (1,2)),
       ((2,0), (2,1), (2,2)),
-      # crossed
+      # diagonal
       ((0,0), (1,1), (2,2)),
       ((2,0), (1,1), (0,2))
     ]
 
     for coordinates in combinations:
-        letters = [board[y][x] for x,y in coordinates]
-        if len(set(letters)) == 1:
+        letters = [board[y][x] for x, y in coordinates]
```

Comment:
When I opened this in learn preview, it's only showing `board[y]`, not board[y][x]. I played around a bit with this and I think the nested square brackets are causing the issue. I played around with this for a bit but could not for the life of me figure out what is causing that behavior and how to fix it. I'm not too concerned as this is in the instructor portion, but I did find it odd and figured I'd point it out. I'm attaching a screenshot of what I see in Learn Preview.
<img width="664" height="95" alt="Screenshot 2025-10-23 at 5 57 12 PM" src="https://github.com/user-attachments/assets/71bd3c5c-f2b7-4be6-b8c5-42f3b383c415" />